packages/demo/src/index.js

```diff
@@ -1,13 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
Do we want to delete this file?

Reply:
we should ignore this deletion from this PR, since it is great place to start when creating new components.

Reply:
No, I was trying to stop tracking my changes to it since I was just using it to test my components. I'll resolve this

Reply:
`git checkout origin/master packages/demo/src/index.js` should do the trick